canonical/canonical.com
# Revert the method of listing departments

## Done
Simply reverted this change: https://github.com/canonical/canonical.com/pull/723/files

## QA
- Open the demo
- Go to careers overview
- Search for a job
- See the search results work
- Click on a role and see the role details render
